app/services/hyrax/fixity_status_service.rb

```diff
@@ -0,0 +1,86 @@
+module Hyrax
+  # Creates fixity status messages to display to user, for a fileset, or
+  # a specific version/file within it. Determines status by looking up
+  # existing ChecksumAuditLog objects, does not actually do a check itself.
+  # See FileSetFixityCheckService and ChecksumAuditLog for actually performing
+  # checks and recording as ChecksumAuditLog objects.
+  class FixityStatusService
+    include ActionView::Helpers::TagHelper
```

Comment:
This to me seems like an odd include for a service object.  I would think that an object that emits HTML would be a presenter.